### PR TITLE
30: Revisit user storage

### LINKED ISSUE #30: Revisit User Storage
- storage interface should not contain methods like „Register“ - rename it
- do we still need the „Create“ method?

### PR DESCRIPTION
**Pull request by GR0ZA**
Closes #30

### CODE REVIEW

File: docs/docs.go
Comment (f-odc) [inline docs/docs.go:1]: Whats happens here?

**Review by f-odc — COMMENTED** [review]
Why are the swagger docs changing?

File: docs/docs.go
Comment (GR0ZA) [reply docs/docs.go:1]: I run make swag to update the docs, apparently I forgot about this in some other PR that is already merged.

That‘s the reason why we need to automatize the process 😄

Comment (f-odc) [reply docs/docs.go:1]: I will create an issue for that. Maybe include swagger in our git pipeline.

**Review by f-odc — APPROVED** [review]

**Pull request merged**